scheduler/clients/javascript/package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "@nettu/sdk-scheduler",
+  "name": "@meetsmore/nittei",
```

Comment:
You should probably use `@nittei` as the org, not `@meetsmore`, since it's an OSS project.

Reply:
Good point, I admit I thought until now that the namespace needed to be the same as the organization on Github, but looks like I'm wrong

I'll rename this to `@nittei/client` when I work on the CI for publishing 👍